app/models/hathitrust.rb

```diff
@@ -189,5 +199,17 @@ def get_hathifile(task)
 
     txt_pathname
   end
-
-end
\ No newline at end of file
+end
+
+
```

Comment:
This commented-out code can be removed if it's not being used.

Reply:
This commented out code has now been removed